src/domain/session/room/timeline/tiles/GapTile.js

```diff
@@ -100,8 +130,11 @@ export class GapTile extends SimpleTile {
 
     get error() {
         if (this._error) {
+            if (this._error instanceof ConnectionError) {
```

Comment:
We don't usually return anything but primitive types from view model properties to keep the view simple. How about also implementing this logic in the isLoading property and just return a string here?